pulp/apis/cplex_api.py

```diff
@@ -303,6 +303,7 @@ def __init__(
             logPath=None,
             epgap=None,
             logfilename=None,
+            threads=0
```

Comment:
i think we use None for the interface of other solvers. It's better to use the same defaults. See

https://github.com/coin-or/pulp/blob/d479cea0edd43eb6f017a39736738fb1008437b7/pulp/apis/cplex_api.py#L15-L40